app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (50, 50, [8, 7]),
+        (24, 24, [2, 2]),
+        (23, 25, [1, 2]),
```

Comment:
The expected result for `get_human_age(23, 25)` should be `[1, 2]` according to the conversion rules. However, the conversion logic for cat years seems incorrect. Based on the rules, 23 cat years should convert to 1 human year, and 25 dog years should convert to 2 human years. Please verify the conversion logic.